### PR TITLE
Feature/developer menu

### PR DESCRIPTION
**Pull request by yoerivanhoof**
This Pull Request (PR) adds functionality to the game according to User Story #XXX

This PR...

* [x] has added functionality.
* [x] complies with the code style of the repository.
* [x] is up to date with the `dev` branch.

Also, I have...

* [x] Updated Taiga.

Please merge with care!

### CODE REVIEW

File: memecity.engine/MemeEngine.h [hunk 1]
```diff
@@ -32,32 +32,76 @@ namespace memecity::engine {
 			MultimediaManager* multimedia_manager;
 			InputManager* input_manager;
 			sdl::TimerFacade* timer;
+			MemeEngine* engine;
```
Comment (simson0606) [inline memecity.engine/MemeEngine.h:35]: Waarom een pointer naar de engine in de engine?

File: memecity.game/Game/States/DeveloperMenuState.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
+﻿#include "DeveloperMenuState.h"
+#include "UI/Menu/MenuBuilder.h"
+#include "../../Assets.h"
+#include "../Input.h"
+#include "../Components/PlayerComponent.h"
+#include "../Components/HealthComponent.h"
+#include "../Systems/ExpSystem.h"
+//todo include state for cheat
```
Comment (simson0606) [inline memecity.game/Game/States/DeveloperMenuState.cpp]: comment...

File: memecity.engine/MemeEngine.h (hunk 1 above)
Comment (yoerivanhoof) [reply memecity.engine/MemeEngine.h:35]: zit niet in de engine, maar in de context

File: memecity.game/Game/States/DeveloperMenuState.cpp (hunk 2 above)
Comment (yoerivanhoof) [reply memecity.game/Game/States/DeveloperMenuState.cpp]: fixed

**Review by simson0606 — APPROVED** [review]

**Pull request merged**